src/domain/hanke/hankeIndexes/HankeIndexes.tsx

```diff
@@ -123,43 +123,43 @@ const HankeIndexes: React.FC<React.PropsWithChildren<Props>> = ({
         <IndexSection
           title={t('hankeIndexes:pyorailynPaareitti')}
           content={`${t('hankeIndexes:kiertoreittitarve')}: ${t(
-            getDetourNeedByIndex(pyorailyIndeksi),
+            getDetourNeedByIndex(pyoraliikenneindeksi),
           )}`}
-          index={pyorailyIndeksi}
-          testId="test-pyorailyIndeksi"
+          index={pyoraliikenneindeksi}
+          testId="test-pyoraliikenneindeksi"
           loading={loading}
           showIndexText={!small}
         />
 
         <IndexSection
           title={t('hankeIndexes:joukkoliikenneRaitiovaunu')}
           content={`${t('hankeIndexes:kiertoreittitarve')}: ${t(
-            getDetourNeedByIndex(raitiovaunuIndeksi),
+            getDetourNeedByIndex(raitioliikenneindeksi),
           )}`}
-          index={raitiovaunuIndeksi}
-          testId="test-raitiovaunuIndeksi"
+          index={raitioliikenneindeksi}
+          testId="test-raitioliikenneindeksi"
           loading={loading}
           showIndexText={!small}
         />
 
         <IndexSection
           title={t('hankeIndexes:joukkoliikenneLinjaauto')}
           content={`${t('hankeIndexes:kiertoreittitarve')}: ${t(
-            getDetourNeedByIndex(linjaautoIndeksi),
+            getDetourNeedByIndex(linjaautoliikenneindeksi),
           )}`}
-          index={linjaautoIndeksi}
-          testId="test-linjaautoIndeksi"
+          index={linjaautoliikenneindeksi}
+          testId="test-linjaautoliikenneindeksi"
           loading={loading}
           showIndexText={!small}
         />
 
         <IndexSection
           title={t('hankeIndexes:ruuhkautuminen')}
           content={`${t('hankeIndexes:kiertoreittitarve')}: ${t(
-            getDetourNeedByIndex(perusIndeksi),
+            getDetourNeedByIndex(autoliikenneindeksi),
```

Comment:
Is it possible to rename the translation hankeIndexes:ruuhkautuminen to for example hankeIndexes:autoliikenne? This can be ignored if the change as unwanted side effects.

Reply:
Yes, it is possible. I did that and also renamed and adjusted texts (from Figma) for other hankeIndexes:
```
    "autoliikenne": "Autoliikenteen ruuhkautuminen",
    "pyoraliikenne": "Pyöräliikenteen merkittävyys",
    "raitioliikenne": "Joukkoliikenteen merkittävyys: Raitioliikenne",
    "linjaautoliikenne": "Joukkoliikenteen merkittävyys: Linja-autojen paikallisliikenne",
```
I removed these from en and sv translations because a) there are no translations afaik and b) raitioliikenne and linjaautoliikenne were already missing in en and sv.